src/TodoListItem.js

```diff
@@ -1,8 +1,13 @@
 import React from "react";
 
-function TodoListItem({ todo }) {
+function TodoListItem({ todo, onRemoveTodo }) {
+
+    function onClick(){
+        onRemoveTodo(todo.id);
+    }
+
```

Comment:
It's good practice to specify the kind of "type" this button is. Even tho it is `type="button"`.